Tools/scripts/update_wiki.py

```diff
@@ -87,5 +87,5 @@ def find_post_by_title(title):
 if r == True:
     print("Upload OK")
     sys.exit(0)
-print 'result: ', r
+print('result: ', r)
```

Comment:
I think this doesn't work for Python2, does it?

Reply:
You are correct.  `from __future__ import print_function`